src/script/media/MediaDevicesHandler.ts

```diff
@@ -272,16 +272,16 @@ export class MediaDevicesHandler {
        * for further info please visit:
        * https://www.electronjs.org/docs/latest/breaking-changes#removed-desktopcapturergetsources-in-the-renderer
        */
-      if (window.desktopCapturer.getDesktopSources) {
-        return window.desktopCapturer.getDesktopSources(options);
+      if (window.desktopCapturer?.getDesktopSources) {
+        return window.desktopCapturer?.getDesktopSources(options);
```

Comment:
I don't think `?.` is needed in 2nd line, since we know this function exists.
```suggestion
      if (window.desktopCapturer?.getDesktopSources) {
        return window.desktopCapturer.getDesktopSources(options);
```